.github/workflows/ci.yml

```diff
@@ -0,0 +1,134 @@
+## (5b) Pin to commits, not tags
+
+name: CI Build and Check
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+env:
+  IMAGE: rancher/kube-api-auth
+
+permissions:
+  contents: read
+  security-events: write # upload Trivy Sarif results
+
+jobs:
+  linter:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Get Sources
+      uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v5
+      with:
+        go-version: '1.22'
```

Comment:
Maybe we can derive the version from the `go.mod`, or export a common variable used in the various setup-go actions: https://github.com/actions/setup-go?tab=readme-ov-file#getting-go-version-from-the-gomod-file

Reply:
Not sure why you see this as needed

Reply:
Ok. Beginning to see. Less places to edit when bumping go version.
Done with commit 0e8291db60e4fdf5b3a737402aa08321450f2b16